use-timescale/data-tiering/enabling-data-tiering.md

```diff
@@ -11,32 +11,60 @@ cloud_ui:
 
 import TieredStorageBilling from "versionContent/_partials/_tiered-storage-billing.mdx";
 
-# Manage automated and manual tiering
+# Manage storage and tiering
 
-You use tiered storage to save on storage costs. Specifically, you can migrate rarely used data from 
-Timescale's standard high-performance storage to the object storage. After you 
+You use high-performance storage for frequently accessed data. You can [change the high-performance storage type][change-storage-type] in $CONSOLE to make sure the available storage and IOPS meet your needs. 
+
+You use low-cost object storage to cut costs by migrating rarely used data from the high-performance storage. After you 
 [enable tiered storage][enable-tiered-storage], you then either [create automated tiering policies][tiering-policies] 
 or [manually tier and untier data][manual-tier].
 
 You can query the data on the object storage tier, but you cannot modify it. Make sure that you are not tiering data that needs to be **actively modified**.
 
 <TieredStorageBilling />
 
+## Change high-performance storage type
```

Comment:
Hmmm I'm finding this whole section kinda confusing. It seems to be all about Enhanced Storage. But most people will be on this page looking to learn about Standard Storage. Why not have one heading for Standard Storage and its experience, and another heading for Enhanced Storage and its experience?

Reply:
Only the first section describes how to go from standard to enhanced and back. The rest of the page explains setting up policies and other things that are the same for both storage types. If we make different headings, 90% of the content will be duplicated. I can maybe move this section down a little bit and place it after "Enable tiered storage", so that we don't talk about enhanced straight away?